docfx_yaml/extension.py

```diff
@@ -1260,6 +1261,24 @@ def find_markdown_pages(app, outdir):
             })
 
 
+# Looks for cross references to look for in given content, except for
+# current_name.
+def find_cross_references(content, current_name, keyword_map):
```

Comment:
"Looks for" and "find" do not sound like the content will actually be modified and returned with cross references replaced.